### ISSUE TITLE
[fix] ComboBox is missing status text and `aria-activedescendant` attribute

### ISSUE THREAD

**Issue by suzubara** [trussworks/react-uswds#1690]
**ReactUSWDS Version & USWDS Version:**

This pertains to the ComboBox component at all versions. Current versions at time of writing are ReactUSWDS 2.2.0 and USWDS 2.11.2.

**Describe the bug**

The ComboBox component is missing two features:
- The status text element should be updated with text contents when the list is opened, filtered, selected, and closed. (see references to `selectEl` in the [source code](https://github.com/uswds/uswds/blob/develop/src/js/components/combo-box.js))
- The input element should have an `aria-activedescendant` attribute set when an option is focused or the list is closed (see references to `aria-activedescendant` in the [source code](https://github.com/uswds/uswds/blob/develop/src/js/components/combo-box.js))
